src/main/java/com/hedvig/contract/customerio/Sweden.kt

```diff
@@ -65,13 +66,13 @@ class Sweden(
     this.updateContractInfo(this.contract, eventId)
   }
 
-  override fun onContractRenewalQueued(renewalDate: LocalDate) {
-    notificationService.sendContractRenewalQueued(
-      contractId = this.contract.id,
-      renewalDate = renewalDate,
-      eventId = this.eventId,
-      memberId = this.contract.holderMemberId
-    )
+  override fun onContractRenewalQueued(contract: ContractState<AgreementState>?) {
+    if (contract != null) {
+      val data = mapOf("renewal_date" to contract.renewal!!.renewalDate.toString())
+      notificationService.updateCustomer(contract.holderMemberId, data)
+    } else {
+      logger.error("Could not find a contract for event ContractRenewalQueued")
+    }
```

Comment:
I prefer making the null check early
```suggestion
  override fun onContractRenewalQueued(contract: ContractState<AgreementState>) {
      val data = mapOf("renewal_date" to contract.renewal!!.renewalDate.toString())
      notificationService.updateCustomer(contract.holderMemberId, data)
```